routes/entryRoutes.js

```diff
@@ -115,4 +118,31 @@ routes.GETstudentEntries = function(req, res){
   });
 }
 
+routes.GETcohortEntries = function(req, res) {
+  var cohort = req.params.cohort;
+  var studentListFull = [];
+  var attendanceList = [];
+  var starsList = [];
+  var datesList = [];
+  var behaviorList = [];
+  var warningList = [];
+  var engageContentList = []
+
+  Student.find({program:cohort}, function(err, students){
+    var studentIds = students.map(function(student){return student._id});
+    console.log('studentIds: '+studentIds);
+    FormDB.find({_studentID:{$in:studentIds}}, function(err, forms){
+      forms.reduce(function(prev,form){
+        attendanceList.push(form.attendance);
+        starsList.push(form.stars);
+        datesList.push(form.date);
+        warningList.push(form.warnings);
+        engageContentList.push(form.engageContent)
+      }, {})
```

Comment:
ah this makes sense. any reason why this whole thing can't be accomplished with a forEach?